task.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/containerd/containerd/mount"
 	"github.com/containerd/containerd/plugin"
 	"github.com/containerd/containerd/rootfs"
+	_ "github.com/containerd/containerd/windows/hcsshimopts"
```

Comment:
What is this for?

Reply:
This was to register the type `ProcessDetails` on the client side, to be used in line 319 `typeurl.UnmarshalAny(p.Info)`

Reply:
Normally `_` imports should be done by the `main` package only. When used in an other package it should be have a comment. Do we want to import this type in linux builds as well? I guess you could use a linux client to drive a windows containerd daemon..?

Reply:
From what I know this is windows-specific process info (it reflects the information returned by a call to hcsshim grabbing additional process information inside the hcs container).

In this PR I did not implement the additional process information for the linux build. But would be happy to look into it! @mlaventure is more familiar - thoughts?

Reply:
`containerd` is only a local daemon, so unless someone makes a proxy to allow remote access the client would have to be a windows client

Reply:
Confirming since this import will be done out of `main` (I can also do it there? I'm unsure tbh) a comment would be good?